src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,33 @@
+import { ImageComponent } from '../ImageComponent/ImageComponent';
+
+//TODO Temporary image callback function
+const getImageOnClickCallback = (id) => {
+    return () => {
+        console.log(id);
+    }
+}
+//END
```

Comment:
You don't need to declare the //END comment here

Reply:
Done